framework/src/util/SharedLibrary.cpp

```diff
@@ -124,6 +123,21 @@ void SharedLibrary::Load(int flags)
 #endif
 }
 
+void SharedLibrary::Load(const std::unordered_map<std::string, Any>& config)
```

Comment:
I suggest to remove this `Load` overload, we can already pass in int options via `Load(int)`. This keeps the SharedLibrary class simpler.

Reply:
Done.